src/components/object-dump/object-dump-directive.js

```diff
@@ -17,12 +17,17 @@
           }
 
           try {
-            var content = scope.content;
+            var content = scope.content,
+                template = handlebarsService.getTemplate(scope.templateKey);
```

Comment:
Can we break this into two `var` declarations (just to make it consistent with the rest of the code base)